src/components/layers/layers.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="text-center">
+    <span class="uhk__layer-switcher--wrapper" data-title="Layers: ">
+        <button *ngIf="len > 0" #baseButton type="button" class="btn btn-default btn-lg btn-primary" (click)="selectLayer(0)">
```

Comment:
Do we check here and below whether all the 4 layers are present within a keymap? I think we shouldn't because all the 4 layers should always be present in every keymap.